.travis.yml

```diff
@@ -44,6 +49,21 @@ script:
   - "echo 'verify the timeout utility works, including that it exits with status 124 on timeout.'"
   -  "(timeout 2 sleep 10; ret=$?; echo timeout ret=$ret; if [[ $ret -eq 124 ]]; then exit 0; fi; exit $ret)"
 
+  # Sanity check that the 'testscript' cmd seems to function at all.
+  # If all the testscripts fail, this is a good one to troubleshoot first.
+  - cd $GOPATH/src/github.com/dvyukov/go-fuzz
+  - testscript -v testscripts/fuzz_help.txt
+
+  # Run our tests for fuzzing modules.
+  # If multiple modules testcripts fail, probably makes sense to start by troubleshooting
+  # the earliest failing testscript.
+  # TODO: probably makes more sense to move this further down, but place here for now for faster iterations.
+  - testscript -v testscripts/mod_go_fuzz_dep.txt
+  - testscript -v testscripts/mod_outside_gopath.txt
+  - testscript -v testscripts/mod_inside_gopath.txt
+  - testscript -v testscripts/mod_v2.txt
+  - testscript -v testscripts/mod_vendor.txt
```

Comment:
why not use a TestScript test? that way you can also remove the 'go get' of the command.

Reply:
> why not use a TestScript test?

I know that is another approach, and I have used that approach elsewhere (for example, in thepudds/fzgo, thepudds/modvet).

There are few reasons that I consciously chose here to instead use the testscript binary utility over a TestScript test, including: 

1. go-fuzz does not have an ability to timeout after a certain duration, but I want to validate that go-fuzz works, and not just that go-fuzz-build works.  This is solvable in multiple ways, but the way I choose to solve it here was to follow the way I solved it previously for go-fuzz under travis, which is using the `timeout(1)` utility via `exec` in the testscripts. Given I am doing `exec` for go-fuzz, I wanted to be consistent and use `exec` for the other executions as well.

2. Maybe 9 or so months ago, getting go-fuzz to test successfully under travis on Windows ending up taking more time than I would have liked when I first contributed CI support to go-fuzz, and those iterations are slow (Windows support for travis is still beta, Windows usually takes much longer to run once started, Windows under travis seems to maybe have additional rate limit compared to Linux, etc.), and go-fuzz on Windows in some cases has an issue being killed compared to a more typical process. Putting those together, here I was looking to avoid doing more iterations on Windows, and hence wanted to follow the pattern I was doing before using `timeout(1)`.

3. I personally have roughly a 50% success rate on predicting what third-party dependencies Dmitry would be excited about accepting into go-fuzz ;-)  Given that, this was a smaller and more loosely coupled change by using the testscript binary.

Obviously, each of those is all solvable via various other approaches, but that is some of the rationale for why I went with the approach here of using the testscript binary.

Reply:
Gotcha. Hopefully we can move towards a nicer `TestScript` test eventually, but this is a step in the right direction anyway.